docs/cli.md

```diff
@@ -341,6 +341,10 @@ poetry add "requests[security,socks]~=2.22.0"
 poetry add "git+https://github.com/pallets/flask.git@1.1.1[dotenv,dev]"
 ```
 
+{{% warning %}}
+Be aware that some shells (in particular Mac Zsh) may not recognise the package name without quotes - for example `poetry add requests[security,socks]`. In this case try instead with quotes `poetry add "requests[security,socks]"`
+{{% /warning %}}
+
```

Comment:
I don't think this warning is very good -- ZSH will behave the same on all platforms, so qualifying it with macOS is useless at best and FUD at worst. I would suggest instead changing line 339 to `"requests[security,socks]"` and changing this warning to be more generalized:

```suggestion
{{% warning %}}
Some shells may treat square braces (`[` and `]`) as special characters. It is suggested to always quote arguments containing these characters to prevent unexpected shell expansion.
{{% /warning %}}

```

Reply:
Good idea.  Much clearer.  Done.  Thanks for the suggestion